src/Interface.js

```diff
@@ -22,7 +22,7 @@ export var struct = Utility.extend(function document () {
 	setAttribute: {value: self},
 	removeAttribute: {value: self},
 	style: {value: {setProperty: self}}
-})
+}, null)
```

Comment:
why `null` here, but `{}` in `src/Exception.js `?

Reply:
This is related to another change that i'm still holding of on, that is – when exception is passed as props to a component it's possible for it to come into a situation where it is not an instance of Object when it should be. Related: https://github.com/dyo/dyo/issues/99#issuecomment-529794312